pkg/tfvars/openstack/openstack.go

```diff
@@ -211,6 +225,7 @@ func TFVars(
 		MachinesNetwork                   string                            `json:"openstack_machines_network_id,omitempty"`
 		MasterAvailabilityZones           []string                          `json:"openstack_master_availability_zones,omitempty"`
 		MasterRootVolumeAvailabilityZones []string                          `json:"openstack_master_root_volume_availability_zones,omitempty"`
+		ExternalLoadBalancer              bool                              `json:"openstack_external_loadbalancer,omitempty"`
```

Comment:
Oh, I see. You have a typo, `openstack_external_loadbalancer` vs `openstack_external_load_balancer`.